cmake/bintools/gnu/target_memusage.cmake

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Add and/or prepare print of memory usage report
+#
+# Usage:
+#   toolchain_bt_output_print_mem_usage(
+#     RESULT_CMD_LIST    <List of commands to be executed, usually after build>
+#     RESULT_BYPROD_LIST <List of command output byproducts>
+#   )
+#
+macro(toolchain_bt_output_print_mem_usage)
+  cmake_parse_arguments(
+    # Prefix of output variables
+    TOOLCHAIN_BT_O_P_MEM_USAGE
```

Comment:
Don't use O_P as a shorthand for OUTPUT_PRINT.

Reply:
So here you would rather have the prefix named TOOLCHAIN_BINARYTOOLS_OUTPUT_PRINT_MEM_USAGE?
Or can we here settle for TOOLCHAIN_BT_OUTPUT_PRINT_MEM_USAGE?